sbncode/CAFMaker/FillReco.cxx

```diff
@@ -548,6 +548,7 @@ namespace caf
 
   void FillTrackCRTHit(const std::vector<art::Ptr<anab::T0>> &t0match,
                        const std::vector<art::Ptr<sbn::crt::CRTHit>> &hitmatch,
+                       const std::vector<art::Ptr<sbn::crt::CRTHitT0TaggingInfo>> &hitmatchinfo,
```

Comment:
The implementation strictly requires one `hitmatchinfo` element.
If this is necessary, then the argument should be a pointer rather than a vector of pointers. If instead this is left more general for future extensions, I request a comment on why if there is more than one hit no information is saved, and that reason should be documented in the code too.
At the moment it seems that the output data record accommodates only one matched hit though.